scripts/package

```diff
@@ -39,3 +39,18 @@ if [ ! -d build/system-charts ]; then
     git clone --branch $SYSTEM_CHART_DEFAULT_BRANCH https://github.com/rancher/system-charts build/system-charts
 fi
 TAG=$TAG REPO=${REPO} go run ../pkg/image/export/main.go build/system-charts $IMAGE $AGENT_IMAGE
+
+mkdir -p ./build/installer/bin
+rsync -a ./rancherd ./build/installer/bin/
+rsync -a ../cmd/rancherd/installer/ ./build/installer/
+makeself.sh \
```

Comment:
So I run `./scripts/package` from the host quite a bit for development.  Can we gate this with a check that makeself.sh exists.  If it doesn't don't run this?  Since in development you most likely won't have makeself installed on your host.

Reply:
Yeah, I can skip this entire block if makeself.sh isnt available on the path. Alternatively we could install it into `bin/build/tools` or the like.

Reply:
Resolved by https://github.com/rancher/rancher/commit/39724451fe6d744777ca4dc458e2654224b03598